addons/product/models/product.py

```diff
@@ -284,6 +284,12 @@ def _compute_product_template_attribute_value_ids(self):
                 ('product_tmpl_id', '=', product.product_tmpl_id.id),
                 ('product_attribute_value_id', 'in', product.attribute_value_ids.ids)])
 
+    @api.multi
+    def is_first_possible_combination(self):
+        self.ensure_one()
+        first_possible_combination = self.product_tmpl_id.get_first_possible_combination().filtered(lambda v: v.attribute_id.create_variant != 'no_variant')
+        return first_possible_combination == self.product_template_attribute_value_ids.filtered(lambda v: v.attribute_id.create_variant != 'no_variant')
```

Comment:
@seb-odoo 

self.product_template_attribute_value_ids cannot contain attributes that are 'no_variant' by definition.
I think you can safely remove the filtering.

Reply:
This method has been removed since.